inquire/src/prompts/select/action.rs

```diff
@@ -50,7 +50,7 @@ impl InnerAction for SelectPromptAction {
             Key::End => Self::MoveToEnd,
 
             key => match InputAction::from_key(key, &()) {
-                Some(action) => Self::FilterInput(action),
```

Comment:
I think the action itself should still be called *Filter*, scoring feels to me like the implementation detail of how we implement the UI feature that is filter input

Reply:
(the new term makes sense everywhere else to me)

Reply:
Yeah, I was unsure on the renaming, I even had to concede to using filter in some of the comments.

I will undo the action rename :+1: